indra/newview/llviewermenu.cpp

```diff
@@ -9779,6 +9808,9 @@ void initialize_menus()
     view_listener_t::addMenu(new LLAdvancedToggleFeature(), "Advanced.ToggleFeature");
     view_listener_t::addMenu(new LLAdvancedCheckFeature(), "Advanced.CheckFeature");
 
+    view_listener_t::addMenu(new LLAdvancedToggleExperiment(), "Advanced.ToggleExperiment");
+    view_listener_t::addMenu(new LLAdvancedCheckExperiment(), "Advanced.CheckExperiment");
+
```

Comment:
I'm adding an "Experimental Features" category in the Advanced menu.  So far it has only one item: "GameControl".